gigascience/gigadb-website
# New feature/1539 fix outstanding issues with public inner pages

# Pull request for issue: #1539

This is a pull request for the following functionalities:

* Handle remiaining a11y issues in the following pages:

http://gigadb.gigasciencejournal.com:9170/site/contact
http://gigadb.gigasciencejournal.com:9170/site/help
http://gigadb.gigasciencejournal.com:9170/site/guide
http://gigadb.gigasciencejournal.com:9170//site/guidegenomic
http://gigadb.gigasciencejournal.com:9170/site/faq
http://gigadb.gigasciencejournal.com:9170/site/team
http://gigadb.gigasciencejournal.com:9170/site/advisory
http://gigadb.gigasciencejournal.com:9170//site/term
http://gigadb.gigasciencejournal.com:9170/site/login
http://gigadb.gigasciencejournal.com:9170/site/forgot

## How to test?

* Navigate to each page
* Run the WAVE audit (chrome extension) and check that there are no "errors" reported
* Verify that the pages look ok
* a11y tests should pass:
```
cd playwright
npm i
npm test
```


## How have functionalities been implemented?

The following is a list of changes:

http://gigadb.gigasciencejournal.com:9170/site/contact

* Fixed focus styles for textarea and button
* Hide "*" and hide "fields with * are required" from screen readers as it doesn't help

http://gigadb.gigasciencejournal.com:9170/site/help

* Remove "-" from dataset icons
* Fix typo
* turn a series of bullet points into a html list

http://gigadb.gigasciencejournal.com:9170/site/guide

* Shorten unnecessary text visible to screen readers from table headers, as they get spelled out for every cell

http://gigadb.gigasciencejournal.com:9170//site/guidegenomic

* Fix missing table borders in Safari
* Add visual indication to currently active dropdown link item

http://gigadb.gigasciencejournal.com:9170/site/advisory

* Give link styling to links that looked like normal text

http://gigadb.gigasciencejournal.com:9170//site/term

* remove bottom border from tabs as it doesn't look ok

http://gigadb.gigasciencejournal.com:9170/site/login

* fix broken aria-describedby references

http://gigadb.gigasciencejournal.com:9170/site/forgot

* Improve form layout and align elements

## Any issues with implementation?

Describe any problems with your implementation

## Any changes to automated tests?

Describe any automated tests that have been developed for the new 
functionalities

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
